edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+## WCA Disciplinary Crash Course
```

Comment:
The title of the document should have only one `#`, to indicate the highest level section title.